src/libs/__mocks__/state-history.js

```diff
@@ -0,0 +1,5 @@
+export const get = () => ({})
```

Comment:
Can we not just auto-mock this file?

Reply:
No, I need `get` to return an empty map.

Reply:
If we leave the other methods blank in this file, will they get automocked?

Reply:
Seems like no.